builtin/credential/ldap/path_config_rotate_root_test.go

```diff
@@ -106,17 +107,20 @@ func TestRotateRootWithRotationUrl(t *testing.T) {
 		Storage:   store,
 	}
 
-	// this should error because the rotation URL is not pointing to anything, which validate the rotation URL is used
 	_, err = b.HandleRequest(ctx, req)
-	if err == nil {
-		t.Fatalf("expected an error when rotating root with a rotation URL that does not point to a valid LDAP server")
+	if err != nil {
+		t.Fatalf("failed to rotate password: %s", err)
 	}
-	// validate that rotationURL doesnt affect LDAP URL in the config
+
 	newCFG, err := b.Config(ctx, req)
-	if err != nil {
-		t.Fatalf("failed to get config after rotation: %s", err)
+	if newCFG.BindDN != cfg.BindDN {
+		t.Fatalf("a value in config that should have stayed the same changed: %s", cfg.BindDN)
+	}
```

Comment:
Maybe we could change this to state the difference more obviously, like `t.Fatalf("BindDN %q changed unexpectedly, found new value %q", cfg.BindDN, newCFG.BindDN)`

Reply:
corrected